dashboard/src/features/shared/components/SearchBar/SearchBar.scss

```diff
@@ -76,9 +75,26 @@
     cursor: pointer;
   }
 }
+.viewdocs {
+  background-image: url('assets/images/info.png');
+  background-repeat: no-repeat;
+  background-size: 15px 15px;
+  background-position: 5px center;
+  display: inline-block;
+  padding: 4px 5px 4px 28px;
+  margin-left: 18px;
```

Comment:
If the goal is to match the spacing of the maginifying glass and placeholder text above, these sizes don't add up. The exact sizes are:

```
  background-position: 0px center;
  padding: 4px 25px;
  margin-left: 21px; // 20px + 1px border from the input element
```

This lets us align the `i` icon from the left edge of the magnifying glass explicitly.